.circleci/config.yml

```diff
@@ -102,10 +106,37 @@ jobs:
     environment:
       PROVIDER: <<parameters.provider>>
       ARGS: <<parameters.args>>
+      IS_UPDATE: "false"
+    steps:
+      - checkout
+      - ci-dependencies
+      - ci-install
+      - ci-test
+      - run:
+          command: ci/uninstall.sh
+          no_output_timeout: 90m
+          when: always
+  ci-update-downgrade:
```

Comment:
do you mean run `update` and `downgrade` in parallel? I'd have to split them into two jobs...

Reply:
The `ci` (`ci/aws/public`) and `ci-update-downgrade` already run in parallel. One thing we can do to optimize is to split the `ci-update-downgrade` into `ci-update` and `ci-downgrade` jobs because `ci-update-downgrade` is taking almost 3 hours.